trabajo-final/spotify_assistant/src/components/listsearchedtracks.js

```diff
@@ -0,0 +1,65 @@
+  import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Track from './track';
+
+class ListSearchedTracks extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {page: 0, pages: []};
+    this.nextPage = this.nextPage.bind(this);
+    this.prevPage = this.prevPage.bind(this);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.setState({page: 0, pages: []});
+    let i = 0;
+    let sublist = nextProps.searchedTracks.slice(0,5);
+    let pages = []
+    while(sublist.length !== 0){
```

Comment:
una mejor implementación podria ser usar un For ya que en una iteración incremental.